srcpkgs/gscreenshot/template

```diff
@@ -10,6 +10,6 @@ maintainer="Rui Flora <rui.flora@gmail.com>"
 license="GPL-2.0-or-later"
 homepage="https://github.com/thenaterhood/gscreenshot"
 distfiles="https://github.com/thenaterhood/gscreenshot/archive/v${version}.tar.gz"
-checksum=f0cdf81e9ab483e11c04ed27141ac37dd6f24415968bc5f4c874e7b17ca5958d
+checksum=18fb296a8115b1b352fe1760cadc12c3ab2a935600cf6ad26e6cdbde3ec2b7f9
 # doesn't ship any tests
```

Comment:
that's not true anymore, the tarball has tests

Reply:
OK :)

Reply:
Tests are buggy...